libs/core/src/lib/@types/geteventlisteners.d.ts

```diff
@@ -1,10 +1,10 @@
-interface EventListener<K extends keyof ElementEventMap> {
-  type: K;
-  listener: (ev: ElementEventMap[K]) => void;
+interface EventListener {
```

Comment:
Will this update be all right? I tested it locally and didn't notice anything broken.

This was to fix the issue I was seeing since `EventListener` is a default type that is not generic:
`All declarations of 'EventListener' must have identical type parameters.`

Reply:
How about just changing the name of the interface?
I missed the fact that it was already taken (looking back, should really have checked it + assume such a generic name would be taken).

even changing to something like `IEventListener` would solve this. If you have suggestions for a better name - I'm open for suggestions (thought of `EventListenerOptions`, but it's taken - see line 4 here).

Reply:
I think `IEventListener` is fitting. I've updated the name and added the generic typing back. Are there any references to IEventListener in other places in the code that I should also be updating?